streams/src/main/java/org/apache/kafka/streams/processor/internals/StreamThread.java

```diff
@@ -834,8 +834,10 @@ void runOnce() {
             }
         }
 
+        final long pollLatency = advanceNowAndComputeLatency();
```

Comment:
Should we compute this, before we call `taskManager.updateNewAndRestoringTasks()` ? Also, do we need to update `now` after `updateNewAndRestoringTasks()` to compute `processLatency` correctly, below?

Reply:
ack